app/stories/pix-banner.stories.js

```diff
@@ -86,4 +93,13 @@ export const argsTypes = {
       ],
     },
   },
+  canCloseBanner: {
+    name: 'canCloseBanner',
+    description: 'Afficher la croix pour fermer la bannière',
+    type: { name: 'boolean', required: true },
```

Comment:
Je ne crois pas qu'il soit obligatoire :) 
```suggestion
    type: { name: 'boolean', required: false },
```